examples/containers/multi/README.md

```diff
@@ -0,0 +1,46 @@
+
+Each node exposes port `:8091` to the host on port `:8091+i` where `i` is from the container name `node$i`, allowing for debugging or testing (e.g., failing over a node) via the Couchbase Server UI.
+
+## Environment Variables ##
+
+There are various environment variables you can configure:
+- `NUM_CLUSTERS`, the number of clusters to partition the nodes into. Defaults to 3.
+- `NUM_NODES`, the number of nodes to create. Defaults to 3.
+
+- `SERVER_USER`, the admin username to configure for all Couchbase Server nodes. Defaults to "Administrator".
+- `SERVER_PWD`, the password for $SERVER_USER to configure for all Couchbase Server nodes. Defaults to "password".
+
+- `CB_VERSION`, the Couchbase Server version (tag on DockerHub) to run on all nodes. Defaults to enterprise-6.6.3.
+- `NODE_RAM` (in MiB). Defaults to 1024, and is used to calculate service quotas for the Data and Index service (the Query service does not have a quota).
+- `LOAD`, a Boolean denoting whether a very light load should be thrown at the cluster using `cbc-pillowfight`, simulating cluster use. Defaults to `true`.
+- `DEV`, a Boolean denoting whether to change the Grafana dashboard update time interval to 10 seconds (from every hour). NOT to be used in production, for facilitating easier dashboard development only. The development changes are reverted upon `make clean`, and should not be committed to the repository.
+
+## Stopping the example ##
+
+You may run `make clean`or alternatively `examples/containers/multi/stop.sh`.
+
+## Grafana Dashboard development ##
+
+The dashboards are currently under development. The directory `/microlith/grafana` is mounted in the CMOS Docker container, with any changes appearing upon refreshing the Grafana webpage.
+
+Non-prometheus stats are obtained using the Grafana community plugin [JSON API](https://grafana.com/grafana/plugins/marcusolsson-json-datasource/), which supports both JSONPath and JSONata. The latter is more heavily used, as it is more expressive.
+
+[Documentation for JSONata](https://docs.jsonata.org/overview), [Playground for JSONata](https://try.jsonata.org/) (useful for testing statements as Grafana isn't very helpful for debugging) - also [available on GitHub](https://github.com/jsonata-js/jsonata-exerciser) to use locally.
+[Documentation for JSONPath](https://goessner.net/articles/JsonPath/), [Playground for JSONPath](http://jsonpath.com/)
+
+## Setting up JSON API for existing Grafana instances ##
+
+Follow [the installation instructions](https://grafana.com/grafana/plugins/marcusolsson-json-datasource/?tab=installation), then configure the URL as cbmultimanager's endpoint with subpath `api/v1`, and the correct BasicAuth username/password for the CMOS stack.
```

Comment:
This is good stuff, but maybe this may be better suited for CONTRIBUTING.md? If I were looking for contribution docs I wouldn't look in an example's README (though that may just be me!).